src/Datagrid/DatagridMapper.php

```diff
@@ -118,15 +118,15 @@ public function has(string $key): bool
         return array_keys($this->datagrid->getFilters());
     }
 
-    public function remove(string $key): self
+    public function remove(string $key): BaseMapper
```

Comment:
This is a little problem, I had to change this one because I changed the BaseMapper remove to:

```
abstract public function remove(string $key): self;
```

In php 7.4 is fine to have this DatagridMapper::remove(): self, but prior to 7.4 it was not implemented that way, and you have to use exactly the same type. What is the correct way to deal with this problem? Should we leave `self` here and avoid adding `self` on BaseMapper (use phpdoc instead) or leave it like now

Reply:
Self here and phpdoc in BaseMapper (with a comment to add self when we drop 7.3) seems the best we can have.

Reply:
Done